microsoftgraph/microsoft-graph-toolkit
# updates storybook to 6.4.4

Closes #1097  
Closes #1099
Closes #1089 
Closes #1090 



### PR Type





Refactoring (no functional changes, no api changes)



### Description of the changes
Upgrades storybook to 6.4.4

### PR checklist

### Other information

<img width="1679" alt="Screen Shot 2021-12-03 at 10 52 51 AM" src="https://user-images.githubusercontent.com/12011447/144569075-1c19c1b5-b57d-4f0d-b2aa-f91d2e5fdfe0.png">


### How to test
1. Run `yarn`
2. Run `yarn start:storybook`
3. Observe that **Storybook** loads and renders stories and that the docs/canvas tabs are not showing.
